server/service/integration_mdm_test.go

```diff
@@ -10828,3 +10828,7 @@ func (s *integrationMDMTestSuite) TestZCustomConfigurationWebURL() {
 	applyResp = applyTeamSpecsResponse{}
 	s.DoJSON("POST", "/api/latest/fleet/spec/teams", teamSpecs, http.StatusUnprocessableEntity, &applyResp)
 }
+
+func (s *integrationMDMTestSuite) TestGetManualEnrollmentProfile() {
+	s.downloadAndVerifyEnrollmentProfile("/api/latest/fleet/mdm/manual_enrollment_profile")
```

Comment:
Perhaps for later: should we add more checks to the `downloadAndVerifyEnrollmentProfile` function?

Reply:
absolutely! what do you think is missing?

Reply:
Would be cool to check the actual contents of the profile more, vs. just them existing. If I can bang something out in a few minutes I'l try to add it into this PR